src/components/Swap/Input.tsx

```diff
@@ -142,7 +153,6 @@ export default function Input() {
           </Row>
         </ThemedText.Body2>
       </TokenInput>
-      <Row />
```

Comment:
@zzmp a little nervous about removing this because I can't figure out why it was here in the first place

Reply:
(except if as a shim to make `gap` on the parent do something?)

Reply:
how could you confirm this either way?

Reply:
It's definitely a spacing shim